packages/webapp/public/locales/fr/translation.json

```diff
@@ -1712,102 +1712,99 @@
+      "RETIRE_SUCCESS": "Retrait réussi du capteur",
+      "RETIRE_FAILURE": "Il y a eu une erreur lors du retrait de ce capteur"
     },
     "TEMPERATURE_READINGS_OF_SENSOR": {
-      "TITLE": "MISSING",
-      "SUBTITLE": "MISSING",
-      "Y_AXIS_LABEL": "MISSING",
-      "WEATHER_STATION": "MISSING",
-      "NO_DATA": "MISSING",
-      "AMBIENT_TEMPERATURE_FOR": "MISSING",
-      "LAST_UPDATED_TEMPERATURE_READINGS": "MISSING"
+      "TITLE": "Température du sol",
+      "SUBTITLE": "Température ambiante actuelle élevée et basse: {{high}}{{tempUnit}} / {{low}}{{tempUnit}}",
+      "Y_AXIS_LABEL": "Température en {{tempUnit}}",
+      "WEATHER_STATION": "Station météorologique: {{weatherStationLocation}}",
+      "NO_DATA": "(aucune donnée)",
+      "AMBIENT_TEMPERATURE_FOR": "Température ambiante pour",
+      "LAST_UPDATED_TEMPERATURE_READINGS": "Dernière mise à jour à {{latestReadingUpdate}}"
     },
     "SOIL_WATER_POTENTIAL_READINGS_OF_SENSOR": {
-      "TITLE": "MISSING",
+      "TITLE": "Potentiel hydrique du sol",
       "SUBTITLE": "MISSING",
-      "Y_AXIS_LABEL": "MISSING",
-      "NO_DATA": "MISSING",
-      "LAST_UPDATED_TEMPERATURE_READINGS": "MISSING"
+      "Y_AXIS_LABEL": "potentiel hydrique du sol en {{soilWaterPotentialUnit}}",
+      "NO_DATA": "(aucune donnée)",
+      "LAST_UPDATED_TEMPERATURE_READINGS": "Dernière mise à jour à {{latestReadingUpdate}}"
     },
     "READINGS_PREVIEW": {
-      "TEMPERATURE": "MISSING",
-      "SOIL_WATER_POTENTIAL": "MISSING"
+      "TEMPERATURE": "Température actuelle du sol",
+      "SOIL_WATER_POTENTIAL": "Potentiel hydrique actuel du sol"
     },
     "SENSOR_READING_CHART_SPOTLIGHT": {
-      "TITLE": "MISSING",
-      "CONTENT": "MISSING"
+      "TITLE": "Les séries peuvent être masquées",
+      "CONTENT": "Les capteurs individuels peuvent être affichés ou masqués en cliquant sur leurs étiquettes ici."
     }
   },
   "CROP_STATUS_NON_ORGANIC_MISMATCH_MODAL": {
-    "TITLE": "MISSING",
-    "SUBTITLE": "MISSING"
+    "TITLE": "Vous avez choisi un emplacement non biologique!",
+    "SUBTITLE": "Vous avez dit que vous allez planter une culture biologique dans un endroit non biologique. Voulez-vous commencer?"
   },
   "CROP_STATUS_ORGANIC_MISMATCH_MODAL": {
-    "TITLE": "MISSING",
-    "SUBTITLE": "MISSING"
+    "TITLE": "Vous avez choisi un emplacement biologique!",
+    "SUBTITLE": "Vous avez dit que vous allez planter une culture biologique dans un endroit non biologique. Voulez-vous commencer?"
   },
   "WAGE": {
     "ERROR": "Le salaire doit être un nombre décimal valide et non négatif",
-    "HOURLY_WAGE": "MISSING",
-    "HOURLY_WAGE_RANGE_ERROR": "MISSING",
-    "HOURLY_WAGE_TOOLTIP": "MISSING"
+    "HOURLY_WAGE": "Salaire horaire",
+    "HOURLY_WAGE_RANGE_ERROR": "Le salaire horaire doit être un nombre positif inférieur à 999 999 999 999",
```

Comment:
It looks like there are three extra 9s..?
English: "Hourly Wage must be a positive number less than 999,999,999"

Reply:
Should also be fixed now